bundle/Resources/docbook/edit/core.xsl

```diff
@@ -634,7 +634,14 @@
         </xsl:if>
         <xsl:if test="@xlink:href">
             <xsl:attribute name="data-href">
-                <xsl:value-of select="@xlink:href"/>
+                <xsl:choose>
+                    <xsl:when test="./docbook:ezattribute/docbook:ezvalue[@key='link-suffix'] != ''">
+                        <xsl:value-of select="concat(@xlink:href, ./docbook:ezattribute/docbook:ezvalue[@key='link-suffix'])"/>
+                    </xsl:when>
+                    <xsl:otherwise>
+                        <xsl:value-of select="concat(@xlink:href, 'test')"/>
```

Comment:
`test` ?

Reply:
@emodric Sorry for that, fixed in https://github.com/netgen/site-bundle/pull/16/commits/2dd2871cb49040ab77b1385c0ebedf6f400509d2